Sources/NIOPosix/Pool.swift

```diff
@@ -202,3 +202,117 @@ extension Int {
         self = (self + alignmentGuide) & (~alignmentGuide)
     }
 }
+
+final class PooledMsgBuffer: PoolElement {
```

Comment:
I think we want this to be a struct, not a class.

Reply:
Needed to deallocate ControlMessageStorage

Reply:
Moved controlMessageStorage back to EL.